src/components/PostCard/PostDetails/PostDetails.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
```

Comment:
Linting + putting details render into it's own component so it can be easily swapped with alternate rendering option of the read only editor `PostDetailsContentTipTap`.